docs/gaia/installation.md

```diff
@@ -43,11 +47,11 @@ $ gaiacli version --long
 `gaiacli` for instance should output something similar to:
 
 ```
-cosmos-sdk: 0.31.2-10-g1fba7308
-git commit: 1fba7308fa226e971964cd6baad9527d4b51d9fc
-vendor hash: 1aec7edfad9888a967b3e9063e42f66b28f447e6
+cosmos-sdk: 0.33.0
```

Comment:
This section is wrong. Should be:

```
cosmos-sdk: 0.33.0
git commit: 06f741609bbce747643e9f1f2abf67040ad78e16
vendor hash: decf1ff8b31df2243aa27bcd6e1a40df4d30be81
build tags: netgo ledger
go version go1.11 darwin/amd64
```

Reply:
y

Reply:
7b4104aced52aa5b59a96c28b5ebeea7877fc4f0 is right, @bez

Reply:
Maybe I'm doing something wrong or maybe the tag is different:

```
$ git checkout v0.33.0; make distclean vendor-deps install; gaiacli version --long

cosmos-sdk: 0.33.0
git commit: 06f741609bbce747643e9f1f2abf67040ad78e16
vendor hash: decf1ff8b31df2243aa27bcd6e1a40df4d30be81
build tags: netgo ledger
go version go1.11 darwin/amd64
```

Reply:
```bash
cosmos-sdk: 0.33.0
git commit: 7b4104aced52aa5b59a96c28b5ebeea7877fc4f0
vendor hash: 5db0df3e24cf10545c84f462a24ddc61882aa58f
build tags: netgo ledger
go version go1.12 linux/amd64
```

Try pulling? I think the `v0.33.0` tag was overwritten once.